swagger/v2/prosecution_case.json

```diff
@@ -110,6 +143,11 @@
         "data"
       ]
     },
+    "example_included_query_parameters": {
+      "description": "an example of parameters that can be included in the query",
+      "type": "string",
+      "example": "hearing_summaries,defendants,defendants.offences"
```

Comment:
What are your thoughts on just having one example? Or if you want to indicate that a user can have multiple includes and needs a comma with no space - can we have two short examples? At the moment it runs over the example box length, and I don't think it can be expanded on swaggerUI

Reply:
I don't personally feel like this example necessarily needs to match the example outlined in the description